.github/workflows/deploy_with_labels.yml

```
@@ -0,0 +1,97 @@
+        if: env.LATEST_COMMIT != env.LAST_SUCCESSFUL_COMMIT
+        run: echo "New commits found. Proceeding with build and deployment."
+
+      - name: Set up Docker Buildx
+        if: env.LATEST_COMMIT != env.LAST_SUCCESSFUL_COMMIT
+        uses: docker/setup-buildx-action@v2
+        with:
+          install: true
+
+      - name: Log in to Docker Hub
+        if: env.LATEST_COMMIT != env.LAST_SUCCESSFUL_COMMIT
+        uses: docker/login-action@v2
```

actionlint (1.7.4)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)